sklearn/plot/_confusion_matrix.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from sklearn.metrics import confusion_matrix
+from sklearn.plot import plot_heatmap
+from sklearn.utils.multiclass import unique_labels
+
+
+def plot_confusion_matrix(y_true, y_pred, classes=None, sample_weight=None,
+                          normalize=False,
+                          xlabel="Predicted Label", ylabel="True Label",
+                          title='Confusion matrix', cmap=None, vmin=None,
+                          vmax=None, ax=None, fmt="{:.2f}",
+                          xtickrotation=45, norm=None):
+    """Plot the confusion matrix as a heatmap. A confusion matrix is computed
```

Comment:
This is a very minor neatpick, but sphinx is probably going to have trouble parsing and formatting this docstring properly in the API summary page: docstring are supposed to have a short title on one line, then a longer description on other lines so that the title can be used to summarize the function's behaviour in long lists of functions (such as here: http://scikit-learn.org/stable/modules/classes.html)